network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/profile_directory.html

```diff
@@ -1,11 +1,19 @@
 {% load static %}
 
 <div class="profile-directory">
-  <div class="profile-filters">
+  <div class="fellowships-directory-filter px-2 mb-5">
     {% for label in filters %}
-    <button class="filter-button">{{ label }}</button>
+    <div class="filter-option">
+      <input type="radio" name="program_year" id="fellow-list-program_year-{{label}}" value="{{label}}" {% if forloop.first %}checked{% endif %}>
+      <label for="fellow-list-program_year-{{label}}">{{label}}</label>
+    </div>
     {% endfor %}
   </div>
 
   {% include "./profile_blocks.html" %}
+
+  {% if filters|length > 0 %}
+  <input type="hidden" data-api-endpoint="{{ api_endpoint }}">
```

Comment:
This is...an interesting approach

Reply:
`input` and `meta` are fairly standard, but as a block, rather than page level, `meta` seemed wrong.

It's either that, or make the JS itself a template that django can render and.. ehhhh... no? =(